k6-get-script.js

```diff
@@ -0,0 +1,17 @@
+import http from 'k6/http';
+
+export let  options = {
+  stages: [
+    { duration: '2m', target: 10 },
+    { duration: '2m', target: 100 },
+    { duration: '2m', target: 1000 },
+    { duration: '2m', target: 100 },
+    { duration: '2m', target: 10 },
+  ],
+  rps: 1000,
```

Comment:
rps: 1000 sets the maximum number or requests per second from the k6 tool.  This will cap out the upper limit of what your service might be able to do.  For example, with this you wouldn't be able to tell if your service can actually handle more thank 1k RPS.